doc/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,143 @@
+# Troubleshooting
```

Comment:
just as a possible source of inspiration for this page, I really liked how EKS structured their doc:
https://docs.aws.amazon.com/eks/latest/userguide/troubleshooting.html

basically, the problem/solution sections is a good template for all of the issues; not sure though, that all of the problems described in our doc have a solution/workaround, but it would be nice to think in that direction

Reply:
In our doc, we are only adding logs for unsupported operations. So, they do not have any solutions. It is for users to know that it is expected behaviour.
Although, for some operations we can add solution like `allow-overwrite` and `allow-delete`.

Reply:
While we're starting with unsupported operations, I do like the EKS doc. I think you already have a structure very similar, Ankit, so we're on the right track!